poll/public/js/poll_edit.js

```diff
@@ -1,12 +1,16 @@
 
 function PollEditUtil(runtime, element, pollType) {
     var self = this;
+    var notify;
 
     // These URLs aren't validated in real time, so even if they don't exist for a type of block
     // we can create a reference to them.
     this.loadAnswers = runtime.handlerUrl(element, 'load_answers');
     this.loadQuestions = runtime.handlerUrl(element, 'load_questions');
 
+    // The workbench doesn't support notifications.
+    notify = typeof(runtime.notify) != 'undefined';
```

Comment:
I think you also need to add an `if (notify)`check just above line 20: https://github.com/open-craft/xblock-poll/blob/style_tweaks/poll/public/js/poll_edit.js#L20